edxval/api.py

```diff
@@ -831,37 +823,70 @@ def export_to_xml(video_ids, course_id=None, external=False):
                 for name in ['profile', 'url', 'file_size', 'bitrate']
             }
         )
+    return create_transcripts_xml(video_id, video_el, resource_fs, transcript_dir)
 
-    return create_transcripts_xml(video_ids, video_el)
 
+def create_trancript_file(edx_video_id, language_code, file_format, transcript_dir, resource_fs):
+    """
+    Writes transcript file to file system.
 
-def create_transcripts_xml(video_ids, video_el):
+    Arguments:
+        edx_video_id (str): Video id of the video transcript file is attached.
+        language_code (str): Language code of the transcript.
+        file_format (str): File format of the transcript file.
+        transcript_dir (str): The Directory to store transcript file.
+        resource_fs (SubFS): The file system to store transcripts.
+    """
+    if transcript_dir and resource_fs:
+        transcript_name = u'{transcript_dir}/{video_id}-{language_code}.{file_format}'.format(
+            transcript_dir=transcript_dir,
+            video_id=edx_video_id,
+            language_code=language_code,
+            file_format=file_format
+        )
+        transcript_data = get_video_transcript_data(edx_video_id, language_code)
+        if transcript_data:
+            transcript_content = transcript_data['content']
+            with resource_fs.open(transcript_name, 'wb') as f:
+                f.write(transcript_content)
+
+
+def create_transcripts_xml(video_id, video_el, resource_fs=None, transcript_dir=None):
```

Comment:
Is there any reason for making ` resource_fs=None, transcript_dir=None` optionals?